Source/Core/DolphinWX/X11Utils.cpp

```diff
@@ -38,7 +38,12 @@ void ToggleFullscreen(Display *dpy, Window win)
 	// Send the event
 	if (!XSendEvent(dpy, DefaultRootWindow(dpy), False,
 	                SubstructureRedirectMask | SubstructureNotifyMask, &event))
+	{
 		ERROR_LOG(VIDEO, "Failed to switch fullscreen/windowed mode.");
```

Comment:
I guess this error here is a generic x11 issue. This doesn't say if fullscreen is now enabled or not. To determ if it's enabled, you would have to spend a round-trip to the x server.